vue/components/ui/select-platforme/select-platforme.stories.js

```diff
@@ -0,0 +1,202 @@
+import { storiesOf } from "@storybook/vue";
+import { withKnobs, text, number, boolean, select } from "@storybook/addon-knobs";
+
+storiesOf("Components", module)
+    .addDecorator(withKnobs)
+    .add("Select", () => ({
+        props: {
+            id: {
+                default: text("Dropdown Id", "select-platforme-id")
+            },
+            placeholder: {
+                default: text("Placeholder", "This is a placeholder text")
+            },
+            options: {
+                type: Array,
+                default: () => [
+                    { id: "option_1", text: "A" },

```

Comment:
check if the names are the same as in the checkbox (master) and radio (PR)

Reply:
its {label: "Example", value: "example" }
but I can't change it or it will break stuff that is already using "dropdown-platforme"

Reply:
:(